tests/test_multistore_file.py

```diff
@@ -0,0 +1,40 @@
+"""Unit tests for oauth2client.multistore_file."""
+
+import errno
+import os
+import tempfile
+import unittest
+
+from oauth2client import multistore_file
+
+
+class _MockLockedFile(object):
+
+    def __init__(self, filename_str, error_code):
+        self.filename_str = filename_str
+        self.error_code = error_code
+        self.open_and_lock_called = False
+
+    def open_and_lock(self):
+        self.open_and_lock_called = True
+        raise IOError(self.error_code, '')
+
+    def is_locked(self):
+        return False
+
+    def filename(self):
+        return self.filename_str
+
+
+class MultistoreFileTests(unittest.TestCase):
+
+    def test_lock_file_raises_ioerror(self):
+        filehandle, filename = tempfile.mkstemp()
+        os.close(filehandle)
```

Comment:
Feel free to leave as-is.

> depends on where those authors are coming from

That was my point. "All" authors know what remove means, not all know that unlink is an alias and forcing them to Google it is unnecessary.

> provide some data 

No time to do that, not worth the battle.